pysamstats/test/test_io.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+import logging
+
+from pysam import Samfile
+from pysamstats import load_coverage, stat_coverage
+
+from pysamstats.io import write_hdf5
+import tables
+
+logger = logging.getLogger(__name__)
+debug = logger.debug
+
+
+def write_hdf5_dtype(arg):
+
+    # testing auto dtype determination.
+    dtype, alignment, result, label = arg
+    import tempfile
+
+    # use auto
+    tmp = tempfile.mktemp(suffix=".h5")
```

Comment:
This temp file will need to be deleted after it's been used.  Probably easiest way to make that happen is to use tempfile.NamedTemporaryFile in a with statement, e.g.:

```
with tempfile.NamedTemporaryFile(suffix=".h5") as tmp:
    write_hdf5("coverage", tmp.name, alignment, ...)
    with tables.open_file(tmp.name, mode='r') as h5file:
        assert ...
```